sys/fmt/fmt.c

```diff
@@ -400,6 +415,16 @@ size_t fmt_lpad(char *out, size_t in_len, size_t pad_len, char pad_char)
     return pad_len;
 }
 
+void fmt_to_lower(char *str)
+{
+    while (str && *str) {
+        if (_is_upper(*str)) {
+            *str = _to_lower(*str);
+        }
+        str++;
+    }
```

Comment:
Could you return the size of the string? that would be consistent with the rest of the library.